licenses/admin.py

```diff
@@ -89,6 +93,11 @@ def changeform_view(
         return super().changeform_view(
             request, object_id, form_url, extra_context)
 
+    def add_view(self, request, form_url="", extra_context=None):
+        """Exclude the number field."""
+        self.exclude = ['number']
+        return super().add_view(request, form_url, extra_context)
+
```

Comment:
set exclude to None again

Reply:
[42064fb](https://github.com/gocept/ok_tools/pull/63/commits/42064fb7ea998c770c5183982208915af6c2deae)